### ISSUE TITLE
BCML will not start when using python3.9

### ISSUE THREAD

**Issue by ilag11111** [NiceneNerd/BCML#99]
With python3.9, on startup BCML will absolutely spam the following error message and will be completely unusable.
```
js: {"short":"__init__() got an unexpected keyword argument 'encoding'","error_text":"Traceback (most recent call last):\n  File \"/home/user/.local/lib/python3.9/site-packages/bcml/util.py\", line 1136, in get_installed_mods\n    {\n  File \"/home/user/.local/lib/python3.9/site-packages/bcml/util.py\", line 1137, in <setcomp>\n    BcmlMod.from_info(info)\n  File \"/home/user/.local/lib/python3.9/site-packages/bcml/util.py\", line 315, in from_info\n    return BcmlMod(info_path.parent)\n  File \"/home/user/.local/lib/python3.9/site-packages/bcml/util.py\", line 270, in __init__\n    self._info = json.loads(\n  File \"/usr/lib/python3.9/json/__init__.py\", line 359, in loads\n    return cls(**kw).decode(s)\nTypeError: __init__() got an unexpected keyword argument 'encoding'\n"}
```
**From the python documentation on json.loads:**
`Changed in version 3.9: The keyword argument encoding has been removed.`

**Quick and dirty fix** (_likely improper and may break things_):
Remove all instances of `, encoding="utf-8"` from json.loads functions.  Other similar functions like json.dumps are unaffected by this change.
**Better fix for end users:** Keep using python 3.8.

With this quick and dirty fix BCML starts up and doesn't present any obvious issues.

**Comment by NiceneNerd** [comment]
TBH, I'm surprised that's the only issue with Python 3.9.

Your quick and dirty fix is probably more reliable than you expect. From a little bit of research, it appears the `encoding` argument hasn't actually _done_ anything in several Python versions, so it's probably exactly what needs to be done.

**Comment by NiceneNerd** [comment]
Fixed in 72c4b0f8c3bdb2459747869e3c667b4d1e6fe0b4
